playbooks/katello_provisioning.yml

```diff
@@ -1,6 +1,8 @@
 - hosts: all
   become: true
+  vars:
+    tftp: true
```

Comment:
Best practice is to scope variables based on role. For example, this would become `libvirt_tftp`. Also, we should set a default in defaults/main.yml